ios/ios-aptos-example/README.md

```diff
@@ -3,30 +3,64 @@
 [![Web3Auth](https://img.shields.io/badge/Web3Auth-SDK-blue)](https://web3auth.io/docs/sdk/pnp/ios)
 [![Web3Auth](https://img.shields.io/badge/Web3Auth-Community-cyan)](https://community.web3auth.io)
 
+
 [Join our Community Portal](https://community.web3auth.io/) to get support and stay up to date with the latest news and updates.
 
-This example demonstrates how to use Web3Auth to get create a Aptos account, connect and interact with Aptos chain in iOS.
+This example demonstrates how to use Web3Auth to create an Aptos account, connect to the Aptos chain, and interact with it in an iOS application using the [Web3Auth PnP Swift SDK](https://web3auth.io/docs/sdk/pnp/ios) and the [Aptos Swift SDK](https://github.com/ALCOVE-LAB/aptos-swift-sdk).
 
 ## How to Use
 
-### Download Manually
+### Installation
+
+1. **Download the Example Project**
+
+   ```bash
+   npx degit Web3Auth/web3auth-pnp-examples/ios/ios-aptos-example w3a-ios-aptos
+   ```
+
+2. **Install Dependencies**
+
+   Navigate to the project directory and install dependencies:
+
+   ```bash
+   cd w3a-ios-aptos
+   ```
+
+3. **Open the Project in Xcode**
+
+   Run the project in Xcode:
 
-```bash
-npx degit Web3Auth/web3auth-pnp-examples/ios/ios-aptos-example w3a-ios-aptos
-```
+   ```bash
+   open ios-aptos-example.xcodeproj
+   ```
 
-Install & Run:
+### Setup
 
-```bash
-cd w3a-ios-aptos
-# Run project in Xcode
-```
+1. **Add the Web3Auth PnP Swift SDK**
+
+   You need to include the Web3Auth PnP Swift SDK in your project. Follow the instructions on [Web3Auth iOS SDK documentation](https://web3auth.io/docs/sdk/pnp/ios) to integrate the SDK into your project. This typically involves adding the SDK as a dependency in your `Podfile` or `Package.swift` file.
+
+2. **Add the Aptos Swift SDK**
+
+   Integrate the Aptos Swift SDK by following the instructions on [Aptos Swift SDK GitHub](https://github.com/ALCOVE-LAB/aptos-swift-sdk). This usually involves adding the SDK as a dependency.
+
+3. **Configure Web3Auth**
+
+   Set up Web3Auth with your configuration by following the [Web3Auth iOS SDK documentation](https://web3auth.io/docs/sdk/pnp/ios). 
+
+4. **Configure Aptos Swift SDK**
+
+   Follow the [Aptos SDK Documentation](https://aptos.dev/en/build/sdks/community-sdks/swift-sdk) to initialize and configure the SDK within your application.
```

Comment:
IMO we can skip this too, because to run the examaple, they will not have to do this setup. It can create confusion. We can keep this in guides.